packages/react/src/progress-indicator/ProgressIndicator.test.tsx

```diff
@@ -105,120 +160,111 @@ describe('ProgressIndicator', () => {
 			expect(await axe(container)).toHaveNoViolations();
 		});
 	});
-	describe('With isActive applied', () => {
-		function createList(
-			activeStatus: ProgressIndicatorItemStatus,
-			extraProperties: Partial<ProgressIndicatorItem>
-		): ProgressIndicatorProps {
-			return {
-				items: [
-					{
-						...extraProperties,
-						label: 'Done',
-						status: 'done',
-						isActive: activeStatus === 'done',
-					},
-					{
-						...extraProperties,
-						label: 'Blocked',
-						status: 'blocked',
-						isActive: activeStatus === 'blocked',
-					},
-					{
-						...extraProperties,
-						label: 'Doing',
-						status: 'doing',
-						isActive: activeStatus === 'doing',
-					},
-					{
-						...extraProperties,
-						label: 'Error',
-						status: 'error',
-						isActive: activeStatus === 'error',
-					},
-					{
-						...extraProperties,
-						label: 'Saved',
-						status: 'saved',
-						isActive: activeStatus === 'saved',
-					},
-					{
-						...extraProperties,
-						label: 'Started',
-						status: 'started',
-						isActive: activeStatus === 'started',
-					},
-					{
-						...extraProperties,
-						label: 'To do',
-						status: 'todo',
-						isActive: activeStatus === 'todo',
-					},
-				],
-			};
-		}
-
-		allProgressStatuses.forEach((status) => {
-			const isWhiteListedStatus =
-				activeProgressStatusAllowedList.includes(status);
-
-			describe(`${status} status active`, () => {
-				function runTest(
-					extraProperties: Partial<ProgressIndicatorItem>
-				): void {
-					try {
-						const { container } = renderProgressIndicator(
-							createList(status, extraProperties)
-						);
-						if (isWhiteListedStatus) {
-							expect(container).toMatchSnapshot();
+
+	describe('legacy handlers', () => {
+		describe('With isActive applied', () => {
```

Comment:
No actual changes to the `With isActive applied` tests. Just moved into this legacy block.